src/main/java/com/walkhub/walkhub/domain/calorielevel/domain/repository/CalorieLevelRepository.java

```diff
@@ -2,12 +2,14 @@
 
 import com.walkhub.walkhub.domain.calorielevel.domain.CalorieLevel;
 import java.util.Optional;
-import org.springframework.data.repository.CrudRepository;
+
+import org.springframework.data.domain.Sort;
+import org.springframework.data.jpa.repository.JpaRepository;
 
 import java.util.List;
 
-public interface CalorieLevelRepository extends CrudRepository<CalorieLevel, Long> {
-    List<CalorieLevel> findAllBy();
+public interface CalorieLevelRepository extends JpaRepository<CalorieLevel, Long> {
+    List<CalorieLevel> findAllBy(Sort sort);
```

Comment:
이거 그냥 method명을
```suggestion
    List<CalorieLevel> findAllByOrderByLevelAsc();
```
는 어떤가요?